rust/src/mat.rs

```diff
@@ -1,93 +1,201 @@
 use std::ptr;
 
-use ::{Box3, Result, Vec3, VoxelType};
+use {Box3, Result, Vec3, VoxelType};
 
 #[derive(Debug)]
 pub struct Mat<'a> {
     data: &'a mut [u8],
     pub shape: Vec3,
     pub voxel_size: usize,
-    pub voxel_type: VoxelType
+    pub voxel_type: VoxelType,
+    is_fortran_order: bool,
```

Comment:
Can we make this `pub` and get rid of the getter? Not because it's bad right now, but just for consistency.